fotobox/frontend-app/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM --platform=linux/amd64 node:22 as build
```

Comment:
We should not use dockerhub images as base images.

Following example can probably help you to adopt a Node.js scratch image
https://github.com/IBM/CodeEngine/blob/main/cos-to-sql/Dockerfile